### DISCUSSION TITLE
Has anyone running QNAP QVR Pro via go2rtc behind Frigate?

### DISCUSSION THREAD

**Discussion by Matzebhv** [blakeblackshear/frigate#6384] [Ask A Question] [answered]
Hi folks,
to reduce wifi-load / camera load i have tried to put my QVR-Pro behind frigate. That works with generic rtsp cam in QVR-Pro but the rtsp stream in QVR-Pro stops every 10-15 seconds for about 2 seconds. Before digging into debugging, has anybody running this setup with success?

Regards
Matthias

**Comment by deanpm70** [comment]
Hi Matthias,

I know it's been a while but did you ever get anywhere with this? I've just run into exactly this problem today. I have 4 cameras which I'm feeding into QVR via go2rtc (standalone for now) and a bunch of others that are coming in direct. All 4 cameras coming from go2rtc loose their stream every 60 seconds for 10 seconds then seamlessly recover. The other cameras are completely unaffected.

**Comment by Matzebhv** [comment, marked as answer]
Hi,
i have given up with QVR Pro. Frigate is so good by now that i have completly switched over all functions to frigate.
